### DISCUSSION TITLE
adguard home bugs

### DISCUSSION THREAD

**Discussion by ghost** [AdguardTeam/AdGuardHome#4673] [Q&A]
hi 

i'm using adguard home on my synology nas ds720plus
everything seems to be ok every day at any time
and start 

but
once a week, and then it's today so thursday 12pm from france
i don't knwow why, i have to shutdown  and restart my nas to get internet
i think to put a task to do that every thursday
i'll see next week

but if dr strange can leave my nas :)

if anyone has an idea

thanks for help

**Comment by ainar-g** [comment]
Apologies for the slow response.  If this is still an issue, please inspect the [verbose log][1].

[1]: https://github.com/AdguardTeam/AdGuardHome/wiki/FAQ#verboselog

**Reply by ghost** [reply]
Hi
Me too appologies but i don't know.

By the way, since august, i change my internet services provider and it gave me a static ip and on the box, i can manage easly port forwading.

There was christmas 2 or 3weeks ago, may be a wish came true. ^^
